gr-utils/python/modtool/modtool_rm.py

```diff
@@ -169,3 +166,17 @@ def _run_subdir(self, path, globs, makefile_vars, cmakeedit_func=None):
         ed.write()
         self.scm.mark_files_updated(('%s/CMakeLists.txt' % path,))
         return files_deleted
+
+
+### COMMAND LINE INTERFACE ###
+@click.command('remove', short_help=ModToolRemove().description)
+@ModTool.common_params
+@ModTool.block_name
+def cli(**kwargs):
+    """ Remove block (delete files and remove Makefile entries) """
+    args = DictToObject(kwargs)
+    try:
+        ModToolRemove().run(args)
+    except ModToolException as err:
+        print(err, file=sys.stderr)
+        exit(1)
```

Comment:
this part is really repetitive, consider moving it to base as part of the core/cli spilt